src/main/scala/org/ergoplatform/settings/NodeConfigurationSettings.scala

```diff
@@ -28,10 +28,17 @@ case class NodeConfigurationSettings(stateType: StateType,
                                      mempoolCleanupDuration: FiniteDuration,
                                      rebroadcastCount: Int,
                                      minimalFeeAmount: Long,
-                                     headerChainDiff: Int)
+                                     headerChainDiff: Int) {
+  /**
+    * Whether the node keeping all the full blocks of the blockchain or not.
+    * @return true if the blockchain is pruned, false if not
+    */
+  val fullBlocksPruned: Boolean = blocksToKeep >= 0
```

Comment:
The name sounds like the number of blocks pruned, not as the boolean flag. I suggest `isFullBlocksPruned`

Reply:
done